README.md

```diff
@@ -125,6 +125,14 @@ sender.send(message, { registrationTokens: registrationTokens }, 10, function (e
   if(err) console.error(err);
   else    console.log(response);
 });
+
+// Collect device registration tokens that you couldn't send messages to
+sender.send(message, { registrationTokens: registrationTokens }, function (err, response) {  
+  var failed_tokens = response.results
+    .map((res, i) => !!res.error ? registrationTokens[i] : undefined)
+    .filter(token => !!token);
```

Comment:
Thank you for the time you took to contribute :smile:

Here are my thoughts:

Isn't the code wrong, mapping all responses without errors (where `res.error` is falsy, so `!!res.error == false`) to `undefined`? Shouldn't it be the opposite?

I would prefer using `null` to `undefined` in the code example, too.

In addition, I think the `!!` are adding confusion, as the evaluation would be the same without them...

Other than that, it is a good example :smile: